src/app/components/work-item-detail/work-item-detail.component.ts

```diff
@@ -875,7 +802,11 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
         }
       }
     });
-    this.save(payload);
```

Comment:
We need to emit the emitEditWI event in the unassignUser function as save function is no longer emitting it.